libs/ui/src/lib/hyperlink/hyperlink.tsx

```diff
@@ -0,0 +1,21 @@
+import { ArrowUpRightIcon } from '@heroicons/react/24/solid';
+
+export interface HyperlinkProps {
+  href: string;
+  text: string;
+  isExternal?: boolean;
+}
+
+export function Hyperlink(props: HyperlinkProps) {
+  const { href, text } = props;
+  const isExternal = props.isExternal || false;
+  return isExternal ? (
+    <a href={href} target="_blank" rel="noopener noreferrer">
```

Comment:
spread props here if we want to add `style` of `className`
https://reactjs.org/docs/jsx-in-depth.html#spread-attributes